blueocean-dashboard/src/main/js/util/UrlUtils.js

```diff
@@ -19,6 +19,14 @@ export const buildPipelineUrl = (organization, fullName, tabName) => {
     return tabName ? `${baseUrl}/${tabName}` : baseUrl;
 };
 
+export const buildClassicConfigUrl = (window, pipeline) => {
+    if (pipeline && pipeline.fullDisplayName) {
+        return `${window.jenkinsCIGlobal.rootURL}/job/${pipeline.fullDisplayName.split('/').join('/job/')}/configure`;
```

Comment:
Also, should this be using `fullName` instead of `fullDisplayName`? I was under the impression that the URL pathing in classic was not based off display names.